<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VRenderInterface</class>
 <widget class="QDialog" name="VRenderInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vectorial rendering options</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QCheckBox" name="includeHidden">
     <property name="toolTip">
      <string>Hidden polygons are also included in the output (usually twice bigger)</string>
     </property>
     <property name="text">
      <string>Include hidden parts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cullBackFaces">
     <property name="toolTip">
      <string>Back faces (non clockwise point ordering) are removed from the output</string>
     </property>
     <property name="text">
      <string>Cull back faces</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="blackAndWhite">
     <property name="toolTip">
      <string>Black and white rendering</string>
     </property>
     <property name="text">
      <string>Black and white</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="colorBackground">
     <property name="toolTip">
      <string>Use current background color instead of white</string>
     </property>
     <property name="text">
      <string>Color background</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="tightenBBox">
     <property name="toolTip">
      <string>Fit output bounding box to current display</string>
     </property>
     <property name="text">
      <string>Tighten bounding box</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="topMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <property name="bottomMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="sortLabel">
       <property name="toolTip">
        <string>Polygon depth sorting method</string>
       </property>
       <property name="text">
        <string>Sort method:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sortMethod">
       <property name="toolTip">
        <string>Polygon depth sorting method</string>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>No sorting</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BSP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Topological</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Advanced topological</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>31</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="SaveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>SaveButton</tabstop>
  <tabstop>CancelButton</tabstop>
  <tabstop>includeHidden</tabstop>
  <tabstop>cullBackFaces</tabstop>
  <tabstop>blackAndWhite</tabstop>
  <tabstop>colorBackground</tabstop>
  <tabstop>tightenBBox</tabstop>
  <tabstop>sortMethod</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>SaveButton</sender>
   <signal>released()</signal>
   <receiver>VRenderInterface</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>released()</signal>
   <receiver>VRenderInterface</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
